VCS/1. User Guides/6 連線 Windows 個體：本機為 Windows.md

```diff
@@ -1 +1,48 @@
+---
+tags: Guide, VCS, TW
+title: 連線 Windows：本機為 Windows | en
+GA: UA-155999456-1
+---
 
+{%hackmd @docsharedstyle/default %}
+{%hackmd @docsharedstyle/twccheader-en %}
+
+
+# Connect to Windows instance: local machine as Windows
+
+After finished [Preparation and Obtain connection information](https://man.twcc.ai/@twccdocs/vcs-guide-connect-prerequisite-en), please refer to the following method to connect to the Windows instance ( local machine as Windows).
+
+:::info
+{%hackmd @docsharedstyle/note-en %}
+Please note that the status of the instance must be **`Ready`** before it can be used online.
```

Comment:
[SUG]

The created instance will enter the **`Ready`** state, and it will take some time before you can connect remotely to it.